synapsemonitor/__main__.py

```diff
@@ -14,17 +15,17 @@
 
 def monitor_cli(syn, args):
     """Monitor cli"""
-    filesdf = monitor.monitoring(
+    ids = monitor.monitoring(
         syn,
         args.view_id,
         users=args.users,
         email_subject=args.email_subject,
         days=args.days,
     )
     if args.output:
-        filesdf.to_csv(args.output, index=False)
+        pd.DataFrame(ids, columns=["synapse_id"]).to_csv(args.output, index=False)
     else:
-        print(filesdf.to_csv(index=False))
+        logging.info(pd.DataFrame(ids, columns=["synapse_id"]).to_csv(index=False))
```

Comment:
We might want to think more about whether to use logging for the cli.  Because if we wanted to pipe the results, we might want to consider `sys.stdout.write`

Reply:
If the user wants to capture the output locally, they can always specify the output file name.  I just want to make sure that the user can control any output to standard out from the command line.

Also, it might be more consistent to have the writing to a file controlled by another 'Action' rather than embedded in the cli.

Reply:
Hmmmm. I'm ok with that but it would make it hard to chain command line tools.  Although not sure there will be too much of:

```
synapsemonitor .... | do-command-line-action ?
```

Since `logging.info` returns something that is not really easily parsed due to the timestamp it adds.

Reply:
I see.  I guess I was thinking that we would somehow come up with a generic Action that would allow the user to interface between the list of modified synapse ids and a desired command line action in a standardized format rather than through the pipe.  But maybe the above is simpler

Reply:
Oh I mean I'm totally for having generic actions, but if someone did want to use command line instead, we should have a clean stdout instead of something with INFO:......

Reply:
Got it.  Changed to sys.stdout.write and removed any header so that it is just one synapse_id per row in both the print statement and csv file